README.rst

```diff
@@ -14,10 +14,12 @@
 .. image:: https://img.shields.io/pypi/dm/kwalitee.svg
         :target: https://pypi.python.org/pypi/kwalitee
 
+.. image:: https://badge.waffle.io/inveniosoftware/kwalitee.png?label=Status%3A%20ready%20for%20work&title=Issues%20ready%20for%20work
```

Comment:
Yes, it seems so.  Although I get `?` instead of `2` issues ready for work...

BTW we could add the number of issues in work and in review, but that would seem like an overkill... I considered it enough to advertise the issues that are ready for work (=issues ready for people to pull from).